app/views/posts/show.html.erb

```diff
@@ -1 +1,4 @@
-<h1>This is a list of all the posts for a specific user.</h1>
+<h1>Post details</h1>
+
+<%= render @post, comments: @comments %>
+
```

Comment:
- [ ] Please your user index file should match the given routes `http://127.0.0.1:3000/users/1/posts/5` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)